scripts/playwright-docker.sh

```
@@ -40,4 +40,11 @@ if [[ ! -d "$NODE_MODULES_CACHE_DIR" ]]; then
     run_command 'npm ci'
 fi
 
-run_command "$1"
+# Combine all arguments into a single command
+COMMAND="$1"
+shift
+if [[ $# -gt 0 ]]; then
+    COMMAND="$COMMAND $@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)